build/Dockerfile

```
@@ -121,58 +121,85 @@ CMD /bin/bash
 # STAGE: devnet-base
 #
 # This stage installs Foundry.
-FROM ${BASE_IMAGE} as devnet-base
+FROM golang:${GO_VERSION}-bookworm as devnet-base
 
 # Install system dependencies.
-ARG DEBIAN_FRONTEND=noninteractive
 RUN <<EOF
 set -e
 apt-get update
-apt-get install -y --no-install-recommends \
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     ca-certificates \
     curl \
-    git \
-    jq \
-    wget \
-    xxd
+    git
 EOF
 
 # Install Foundry from downloaded pre-compiled binaries
 ARG FOUNDRY_NIGHTLY_VERSION
 RUN curl -sSL https://github.com/foundry-rs/foundry/releases/download/nightly-${FOUNDRY_NIGHTLY_VERSION}/foundry_nightly_linux_$(dpkg --print-architecture).tar.gz | \
     tar -zx -C /usr/local/bin
 
+
 # STAGE: devnet-builder
 #
 # This stage builds the devnet state that will be loaded in Anvil.
 FROM devnet-base as devnet-builder
 ARG DEVNET_BUILD_PATH
 WORKDIR ${DEVNET_BUILD_PATH}
 
+# Install dependencies needed to install yarn
+RUN <<EOF
```

To avoid hiding errors during build.

```suggestion
RUN <<EOF
set -e
```